src/components/ui/UiLineDescription.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import arrowForward from '@/assets/icons/arrow-forward.svg'
+const props = defineProps({
+  id: Number,
+  name: String,
+  start: String,
+  end: String,
+  frequency: Number,
+})
+</script>
+<template>
+  <div
+    class="box-border flex flex-row items-center px-3 py-0 gap-4 w-96 h-20 border-b border-solid border-b-neutral-300"
```

Comment:
remove

- `flex-row`: it's default
- `box-border & border-solid`: it's redundant